ddl/ddl_worker.go

```diff
@@ -464,3 +476,64 @@ func updateSchemaVersion(t *meta.Meta, job *model.Job) (int64, error) {
 	err = t.SetSchemaDiff(diff)
 	return schemaVersion, errors.Trace(err)
 }
+
+func (d *ddl) cleanAddIndexQueueJobs(txn kv.Transaction) error {
+	startTime := time.Now()
+	m := meta.NewMeta(txn)
+	m.SetJobListKey(meta.AddIndexJobListKey)
+	for {
+		job, err := d.getFirstDDLJob(m)
+		if err != nil {
+			return errors.Trace(err)
+		}
+		if job == nil {
```

Comment:
I am confused since we already did err check before this line. Why do we need check the `job`'s value?